src/instance/InstanceController.ts

```diff
@@ -1301,3 +1302,28 @@ export class InstancesController {
     }
   }
 }
+
+@ApiTags('Clients')
+@Controller('clients')
+export class ClientsController {
+  constructor(private readonly instanceService: InstanceService) {}
+
+  @Delete()
+  @ApiOperation({summary: 'Delete all clients.'})
+  @ApiResponse({
+    schema: {
+      example: {},
```

Comment:
The return example in the old version is wrong
<img width="478" alt="Screenshot 2020-05-05 at 12 33 08 PM" src="https://user-images.githubusercontent.com/125676/81052725-a4592680-8ecc-11ea-9ab6-156e4278f65a.png">


https://github.com/wireapp/wire-web-ets/blob/master/src/routes/instance/mainRoutes.ts#L225